python/lsst/ts/standardscripts/auxtel/daytime_checkout/atpnuematics_checkout.py

```diff
@@ -0,0 +1,163 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["ATPnuematicsCheckout"]
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.auxtel.atcs import ATCS
+from ...utils import get_topic_time_utc
+
+STD_TIMEOUT = 5  # seconds
+
+
+class ATPnuematicsCheckout(salobj.BaseScript):
+    """Pnuematics Checkout SAL Script.
+
+    This script performs the daytime checkout of the Auxiliary
+    Telescope pnuematics system to ensure it is ready to be released
+    for nighttime operations.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    - "Opening pnuematics valves": Before opening pnuematics valves, this will
+    pressurize the system.
+    - "Turning on ATAOS coerrections": Before turning on ATAOS corrections.
+    - "Turning off ATAOS coerrections": Before turning off ATAOS corrections.
+    - "Lowering Mirror to hardpoints": Before lowering mirror to hardpoints.
+    - "Exercising Mirror Covers and Vents": Before opening and closing the
+    mirror covers and vents, final checkpoint in script.
+
+    **Details**
+
+    This script performs a daytime checkout of the Auxiliary Telescope
+    Pnuematics system. It will first turn on the valves and check that the line
+    pressure is sufficient for operations. Then, it will turn on/off the ATAOS
+    corrections before lowering the mirror back down onto its hardpoints.
+    Finally the mirror cover and vents are opened and closed. No telescope
+    movement is performed.
+    """
+
+    def __init__(self, index=1):
+
+        super().__init__(
+            index=index,
+            descr="Execute daytime checkout of AT Pnuematics.",
+        )
+
+        # Instantiate atcs. We need to do this after the call to
+        # super().__init__() above. We can also pass in the script domain and
+        # logger to both classes so log messages generated internally are
+        # published to the efd.
+        self.atcs = ATCS(domain=self.domain, log=self.log)
+        self.main_air_pressure_min_threshold = 275790
+        self.main_air_pressure_max_threshold = 413000
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
+        # This script does not require any configuration
+        pass
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = 5.0 * 60  # Approximate 5min to completion
+
+    async def run(self):
+
+        await self.assert_feasibility()
+
+        await self.checkpoint("Opening pnuematics valves")
+
+        await self.atcs.open_valves()
+        pressure = await self.atcs.rem.atpneumatics.tel_mainAirSourcePressure.next(
+            flush=True, timeout=STD_TIMEOUT
+        )
+        if pressure.pressure > self.main_air_pressure_min_threshold:
```

Comment:
This is still not checking the upper limit.. but maybe it is fine?

Reply:
Added the upper limit to the if statement.